db/db_impl_write.cc

```diff
@@ -148,8 +169,10 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
   // and protects against concurrent loggers and concurrent writes
   // into memtables
 
-  last_batch_group_size_ =
+  uint64_t batch_group_size =
       write_thread_.EnterAsBatchGroupLeader(&w, &write_group);
+  last_batch_group_size_.fetch_add(batch_group_size,
+                                   std::memory_order_relaxed);
```

Comment:
So if the last commit writes 1 bytes while the last prepares have written 1GB to WAL, should not we throttle the writes to WAL?